### PR TITLE
Add CI Visibility custom metrics section to dotnet docs.

### PR DESCRIPTION
**Pull request by tonyredondo**
### What does this PR do?
This PR adds a new section into the CI Visibility DotNet documentation to `Add custom metrics to tests`.

### Motivation
This section was missing.

### Additional Notes
This is a continuation of: https://github.com/DataDog/documentation/pull/18831

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_integration/tests/dotnet.md [hunk 1]
```diff
@@ -148,6 +152,26 @@ if (scope != null) {
...
+// inside your test
+var scope = Tracer.Instance.ActiveScope; // from Datadog.Trace;
+if (scope != null) {
+    scope.Span.SetTag("memory_allocations", 16);
+}
+// test continues normally
+// ...
+```
+
+To create filters or `group by` fields for these tags, you must first create facets. For more information about adding tags, see the [Adding Tags][7] section of the .NET custom instrumentation documentation.
```
Comment (juan-fernandez) [inline content/en/continuous_integration/tests/dotnet.md:173]:
```suggestion
To create filters or visualizations for these tags, you must first create facets. For more information about adding tags, see the [Adding Tags][7] section of the .NET custom instrumentation documentation.
```
I don't think you can group by a measure 

Also, here we should link to the guide here https://github.com/DataDog/documentation/pull/18831

Comment (juan-fernandez) [reply content/en/continuous_integration/tests/dotnet.md:173]: https://github.com/DataDog/documentation/pull/18831 is merged, so you can link to the guide

File: content/en/continuous_integration/tests/dotnet.md [hunk 2]
```diff
@@ -134,6 +134,10 @@ All other [Datadog Tracer configuration][6] options can also be used.
 
 ### Adding custom tags to tests
 
+<div class="alert alert-warning">
+  <strong>Note:</strong> To add custom tags to tests, you must configure <a href="#custom-instrumentation">Custom Instrumentation</a> first.
+</div>
```
Comment (cswatt) [inline content/en/continuous_integration/tests/dotnet.md:139]:
```suggestion
To add custom tags to tests, configure [custom instrumentation](#custom-instrumentation) first.
```

File: content/en/continuous_integration/tests/dotnet.md [hunk 3]
```diff
@@ -148,6 +152,28 @@ if (scope != null) {
 
 To create filters or `group by` fields for these tags, you must first create facets. For more information about adding tags, see the [Adding Tags][7] section of the .NET custom instrumentation documentation.
 
+### Adding custom metrics to tests
+
+<div class="alert alert-warning">
+  <strong>Note:</strong> To add custom metrics to tests, you must configure <a href="#custom-instrumentation">Custom Instrumentation</a> first.
+</div>
```
Comment (cswatt) [inline content/en/continuous_integration/tests/dotnet.md:159]:
```suggestion
To add custom metrics to tests, configure [custom instrumentation](#custom-instrumentation) first.
```

File: content/en/continuous_integration/tests/dotnet.md [hunk 4]
```diff
@@ -148,6 +152,28 @@ if (scope != null) {
...
+if (scope != null) {
+    scope.Span.SetTag("memory_allocations", 16);
+}
+// test continues normally
+// ...
+```
+
+To create filters or visualizations for these tags, you must first create facets. For more information about adding tags, see the [Adding Tags][7] section of the .NET custom instrumentation documentation.
+
+Read more about custom metrics in [Add Custom Metrics Guide][15].
```
Comment (cswatt) [inline content/en/continuous_integration/tests/dotnet.md:175]:
```suggestion
Read more about custom metrics in the [Add Custom Metrics Guide][15].
```

**Review by cswatt — CHANGES_REQUESTED** [review]
looks good, but i don't think we need callouts or a bolded "note" — I think it's sufficient to just  have it as the first line of the section.

**Review by juan-fernandez — APPROVED** [review]

**Comment by tonyredondo** [comment]
We have to wait a dotnet tracer release before merging.

**Comment by tonyredondo** [comment]
The release is out, we can merge this.

**Review by rtrieu — APPROVED** [review]
Looks great!

**Pull request merged**